editor/editor_inspector.cpp

```diff
@@ -2613,16 +2652,20 @@ EditorInspector::EditorInspector() {
 	use_folding = false;
 	update_all_pending = false;
 	update_tree_pending = false;
-	refresh_countdown = 0;
 	read_only = false;
 	search_box = nullptr;
 	keying = false;
 	_prop_edited = "property_edited";
-	set_process(true);
+	set_process(false);
 	property_focusable = -1;
 	sub_inspector = false;
 	deletable_properties = false;
 
 	get_v_scrollbar()->connect("value_changed", callable_mp(this, &EditorInspector::_vscroll_changed));
 	update_scroll_request = -1;
+	if (EditorSettings::get_singleton()) {
+		refresh_countdown = float(EditorSettings::get_singleton()->get("docks/property_editor/auto_refresh_interval"));
+	} else {
+		refresh_countdown = 0.33;
+	}
```

Comment:
In practice I guess it's either/or - does `EditorInspector` get instantiated before `EditorSettings`?

Reply:
problem is doc data which loves to instantiate every class under the sun to fetch the default parameters, so if this is not done it will crash. I will add a comment to explain why this was done.